### PR TITLE
Add hBPF project information

### PR DESCRIPTION
**Pull request by rprinz08**
This pull request adds project information for **hBPF** (an eBPF CPU implemented in hardware on FPGA) to projects page under Emerging Projects section.

### CODE REVIEW

File: src/pages/projects.js [hunk 1]
```diff
@@ -422,6 +422,41 @@ const ProjectDescriptions = () => (
 
     <TitleWithAnchor headerClassName="projects-title projects-common-title">Emerging Projects</TitleWithAnchor>
 
+    <div className="project-box">
```
Comment (lmb) [inline src/pages/projects.js:425]: For now I'd say it's better to append to the list of emerging projects rather than prepend.

Comment (lmb) [reply src/pages/projects.js:425]: Sorry, seems like the list is currently in alphabetical order. wfm.

**Review by lmb — APPROVED** [review]

**Review by dthaler — APPROVED** [review]

**Comment by joestringer** [comment]
Closing and reopening to see if it will trigger the GitHub workflow to get a rendered preview of the page.

File: src/pages/projects.js [hunk 2]
```diff
@@ -422,6 +422,41 @@ const ProjectDescriptions = () => (
...
+      <TitleWithAnchor headerClassName="project-major-title projects-common-title" headerTag="h3">hBPF</TitleWithAnchor>
+      <div className="project-minor-title">
+        hBPF = eBPF in hardware
```
Comment (joestringer) [inline src/pages/projects.js:428]: nit: I understand that this is part of the project tagline, but I wonder whether the `=` gives the wrong impression to folks visiting this page; vendors have been running eBPF in hardware for some time now without this project, so equating these two with the `=` seems definitive without recognising the existance of alternatives. Would you be open to some slighty modification here?

```suggestion
        hBPF - eBPF in hardware
```

**Review by joestringer — COMMENTED** [review]
Thanks for your patience on this PR. Would you be able to rebase this against the latest tree? 
This will help to understand how this project fits into the overall structure given the recent changes. We've recently made some changes to the layout of the website, as well as introducing netlify previews so that we can see what these changes will look like before merging the commits. 

Minor nit below.

File: src/pages/projects.js (hunk 2 above)
Comment (rprinz08) [reply src/pages/projects.js]: I'm fine with using '-' instead of '='.

**Comment by joestringer** [comment]
@dthaler I think you wanted to see the preview on the page to confirm which category it should go under?

**Comment by dthaler** [comment]
I agree this is good to add and is emerging, but shouldn't it be under "Core Infrastructure (Emerging)" instead of "Applications (Emerging)"?

**Comment by rprinz08** [comment]
Hi, is there anything I can do to help finishing this PR? Should I move the entry from "Applications (Emerging)" to "Core Infrastructure (Emerging)"?

**Comment by dthaler** [comment]
> Hi, is there anything I can do to help finishing this PR? Should I move the entry from "Applications (Emerging)" to "Core Infrastructure (Emerging)"?

Yes please do.  We just discussed this briefly in the BSC meeting this week.

**Review by dthaler — APPROVED** [review]
@borkmann please review

**Review by borkmann — APPROVED** [review]
lgtm, thanks!

**Pull request merged**